projects/admin/src/app/acquisition/components/order/order-email-form/order-email-form.component.ts

```diff
@@ -0,0 +1,129 @@
+import { Tools } from '@app/admin/shared/preview-email/utils/tools';
+import { TranslateService } from '@ngx-translate/core';
+import { ToastrService } from 'ngx-toastr';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'admin-order-email-form',
+  templateUrl: './order-email-form.component.html'
+})
+export class OrderEmailFormComponent implements OnInit, OnDestroy {
+
+  /** the related order */
+  @Input() order: IAcqOrder;
+
+  /** Closing event for the modal dialog */
+  @Output() closeDialog = new EventEmitter<boolean>(false);
+
+  /** Reload data event to enable detection of data loading */
+  @Output() recordChange = new EventEmitter<any>();
+
+  /** Available recipient types */
+  emailTypes = ['to', 'cc', 'bcc', 'reply_to'];
+
+  /** Mandatory email types */
+  mandatoryEmailTypes = ['to', 'reply_to'];
+
+  /** Suggested emails and PrePopulate recipients */
+  suggestions: { emails: string[], recipients: ITypeEmail[] } = { emails: [], recipients: []};
+
+  /** preview message data */
+  response?: IAcqOrderPreviewResponse;
+
+  /** all component subscription */
+  private _subscriptions = new Subscription();
+
+  /**
+   * Constructor
+   * @param _acqOrderApiService - AcqOrderApiService
+   * @param _toastrService - ToastrService
+   * @param _translateService - TranslateService
+   */
+  constructor(
+    private _acqOrderApiService: AcqOrderApiService,
+    private _toastrService: ToastrService,
+    private _translateService: TranslateService
+  ) { }
+
+  /** onInit hook */
+  ngOnInit(): void {
+    this._subscriptions.add(this._acqOrderApiService
+      .getOrderPreview(this.order.pid)
+      .subscribe((response: IAcqOrderPreviewResponse) => {
+        this.suggestions = Tools.processRecipientSuggestions(response.recipient_suggestions);
+        this.response = response;
+      })
+    );
+  }
+
+  /** onDestroy hook */
+  ngOnDestroy(): void {
+      this._subscriptions.unsubscribe();
+  }
+
+  /**
+   * Sending the order
+   * @param recipients Array of email addresses
+   */
+  dataSend(recipients: ITypeEmail[]): void {
+    this._acqOrderApiService
+      .sendOrder(this.order.pid, recipients)
+      .subscribe(
+        (notification: Notification) => {
+          if (notification.notification_sent) {
+            this._toastrService.success(
+              this._translateService.instant('order has been sent'),
+              this._translateService.instant('Order sent')
+            );
+          } else {
+            this._toastrService.warning(
+              this._translateService.instant('order not yet send'),
+              this._translateService.instant('Order delayed'),
+              { disableTimeOut: true, closeButton: true }
+            );
+          }
+          this.closeEmailDialog();
+          this._acqOrderApiService
+            .getOrder(this.order.pid)
+            .subscribe((order: IAcqOrder) => this.recordChange.next(order));
```

Comment:
Not sure... just a proposition: 
is it really necessary to reload the order in this component ? Why it's not the calling parent component that reload the order ? your `recordChange` event could only send boolean value